Jenkinsfile

```diff
@@ -100,8 +100,5 @@ pipeline {
     success {
       cleanupAndNotify(currentBuild.currentResult)
     }
-    unsuccessful {
```

Comment:
TODO: This will be restored before merge. I deleted this so we won't receive notifications while I work